### PR TITLE
adding offline_stub_SUITE

### PR DESCRIPTION
**Pull request by DenysGonchar**
mod_offline_stub doesn't require any conversion to support dynamic domains. Just added some simple offline_stub_SUITE to ensure that it's working correctly w/o any changes.

### CODE REVIEW

File: big_tests/tests/offline_stub_SUITE.erl [hunk 1]
```diff
@@ -0,0 +1,71 @@
+%%%===================================================================
+%%% @copyright (C) 2011, Erlang Solutions Ltd.
```
Comment (NelsonVides) [inline big_tests/tests/offline_stub_SUITE.erl:2]: Wrong year, and also, I don't think this copyright header is needed 🤔

File: big_tests/tests/offline_stub_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+suite() ->
+    escalus:suite().
+
+%%%===================================================================
+%%% Init & teardown
+%%%===================================================================
+
+init_per_suite(C) -> escalus:init_per_suite(C).
+end_per_suite(C) -> escalus_fresh:clean(), escalus:end_per_suite(C).
```
Comment (NelsonVides) [inline big_tests/tests/offline_stub_SUITE.erl:26]:
```suggestion
init_per_suite(C) ->
    escalus:init_per_suite(C).
end_per_suite(C) ->
    escalus_fresh:clean(),
    escalus:end_per_suite(C).
```

**Review by NelsonVides — COMMENTED** [review]
Just a couple of comments, ready otherwise.

File: big_tests/tests/offline_stub_SUITE.erl [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+init_per_testcase(Name, C) ->
```
Comment (arcusfelis) [inline big_tests/tests/offline_stub_SUITE.erl:22]: Just use Config, Config1, Config2... for consistency with other code.

**Review by arcusfelis — APPROVED** [review]
ok

**Review by NelsonVides — APPROVED** [review]
👍🏽

**Review by NelsonVides — APPROVED** [review]
Still 👍🏽 🙂

**Review by NelsonVides — APPROVED** [review]
Good, just waiting for CI now.

**Review by NelsonVides — APPROVED** [review]
👍🏽

**Pull request merged**